internal/host/validator.go

```diff
@@ -254,7 +254,7 @@ func (v *validator) printIsMachineCidrDefined(context *validationContext, status
 	case ValidationSuccess:
 		return "Machine network CIDR is defined"
 	case ValidationFailure:
-		return "Machine network CIDR is undefined"
+		return "Machine network CIDR is undefined.  The machine network CIDR is defined by setting either the API vip or the Ingress VIP"
```

Comment:
`CIDR can be defined`

Reply:
No. It explains the procedure how it is defined.  There is no other way.